From 08e048488045f3b8b1cce8584fc9d98aa2971a71 Mon Sep 17 00:00:00 2001
From: Stanislaw Kardach <skardach@marvell.com>
Date: Mon, 10 Feb 2020 21:45:00 +0100
Subject: [PATCH 0363/1921] octeontx2-af: fix compilation warning

Ensure correct formatting in debug prints.

Change-Id: I87e209efea8375a253a503c2490d34bfa5afcfca
Signed-off-by: Stanislaw Kardach <skardach@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/23182
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Stefan Chulski <stefanc@marvell.com>
Reviewed-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 drivers/net/ethernet/marvell/octeontx2/af/rvu_npc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/marvell/octeontx2/af/rvu_npc.c b/drivers/net/ethernet/marvell/octeontx2/af/rvu_npc.c
index eda1fb53529b..c209e3f0636b 100644
--- a/drivers/net/ethernet/marvell/octeontx2/af/rvu_npc.c
+++ b/drivers/net/ethernet/marvell/octeontx2/af/rvu_npc.c
@@ -1151,7 +1151,7 @@ static int npc_apply_custom_kpu(struct rvu *rvu,
 	}
 	/* Verify if profile fits the HW */
 	if (fw->kpus > profile->kpus) {
-		dev_warn(rvu->dev, "Not enough KPUs: %d > %d\n", fw->kpus,
+		dev_warn(rvu->dev, "Not enough KPUs: %d > %ld\n", fw->kpus,
 			 profile->kpus);
 		return -EINVAL;
 	}
-- 
2.31.1

